src/Plugin/Autocomplete/Payload.php

```diff
@@ -135,9 +135,7 @@ protected static function fromSqlRequest(Request $request): static {
 		$self = new static();
 		$self->query = $matches[1];
 		$self->table = $matches[2];
-		if (!empty($matches[3])) {
-			$self->parseOptions($matches[3]);
-		}
+		$self->parseOptions($matches[3] ?? '');
```

Comment:
I don't think this approach is better. You call `parseOptions` for the empty attribute that calls KeyboardLayout inside that reads some file from the filesystem and after you check if this parameter is empty. As a result, we have unnecessary calls and checks.

Reply:
Probably you're right. The default layouts we extracted were set to the wrong layer. Let's double-check that and make sure they're applied correctly.